test/util.ts

```diff
@@ -1214,6 +1214,36 @@ describe('common/util', () => {
         config.shouldRetryFn!();
       };
     }
+    const errorMessage = 'Error.';
+    const customRetryRequestFunctionConfig = {
+      retryOptions: {
+        retryableErrorFn: function (err: ApiError) {
+          if (err.message === errorMessage) {
```

Comment:
Perhaps nitpick-y but this could be made more succinct by just saying `return err.message === errorMessage`.

Reply:
so smart